game/game.py

```diff
@@ -122,8 +114,8 @@ def modify_score(self, event, payload={}):
             self.current_player.update_score(SCORES[CORRECT_MOVE])
         elif event == INVALID_MOVE:
             self.current_player.update_score(SCORES[INVALID_MOVE])
-        elif event == TIMEOUT:
-            self.current_player.update_score(SCORES[TIMEOUT])
+        elif event == TIMEOUT_SC:
```

Comment:
couldn't you simplify this logic by doing something like this?

```
else: 
       self.current_player.update_score(SCORES[event])
```

Reply:
Implemented as suggested now.